.changeset/quiet-icons-think.md

```diff
@@ -0,0 +1,15 @@
+---
+'ember-scoped-css': minor
+---
+
+Support `gjs` and `gts` (`<template>`) by allowing an import for `scopedClass`.
+
+Example usage:
+
+```gjs
+import { scopedClass } from 'ember-scoped-css';
+
+<template>
+  <p class={{scopedClass "hi"}}>text color</p>
```

Comment:
Do you _need_ to do this? Or is it just allowed now?

Reply:
we current use `scoped-class` in the template, and that works, but we have to disable linting and type checking in the template to do it.

This changes allows us to keep linting / type checking enabled.